### DISCUSSION TITLE
*NEW* Tag for Scripts

### DISCUSSION THREAD

**Discussion by yodaphone** [tteck/Proxmox#822] [Ideas]
A quick suggestion

You're scripts are awesome & i use them extensively. thank you for that. If i may, can you please add a *NEW* tag to scripts that have been published/updated recently?

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/blob/main/CHANGELOG.MD
This doesn't suffice?

![changelog](https://user-images.githubusercontent.com/21342670/206701383-ac336899-6cfe-4013-9096-df9d302cdbfa.png)

**Comment by yodaphone** [comment]
there is typically a tendency to ignore the changelog. a small "NEW" icon next to the script title might be more "eye catching"

**Reply by tteck** [reply]
IMHO, a changelog is enough.

**Reply by techbitsio** [reply]
Looking at the changelog, the latest change is the addition of Change Detection (https://changedetection.io/). This would be a good choice to track changes to the changlog. Change.
